tests/unit/editors/textEditor/tests.html

```diff
@@ -1575,6 +1575,18 @@
 			ok($('#maxLengthTest').parents().hasClass("ui-ignotify-warn"), "The notifier is not shown");
 			$editor.remove();
 		});
+		
```

Comment:
This test is not for that issue. This test was for #481 , but was not merged to 17.1 and 17.2. The test for this issue is pull request #1110